ebu_tt_live/bindings/validation/timing.py

```diff
@@ -265,6 +275,17 @@ def _semantic_manage_timeline(self, dataset, element_content):
         # Register on timeline
         doc.add_to_timeline(self)
 
+    def _semantic_pushing_computed_time(self, dataset):
+        if dataset['timing_syncbase'] is not None and dataset['timing_syncbase'] > timedelta(0):
+            if self.begin is not None and self.end is not None:
+                self._pushed_computed_begin_time = self.computed_begin_time
+                self._pushed_computed_end_time = self.computed_end_time
+            elif self.begin is None or self.end is None:
```

Comment:
the conditional in this `elif` is redundant - it would be the same result if it were just an `else`